resources/imports.js

```diff
@@ -0,0 +1,25 @@
+function reference(link) {
+	const headerContent = `
```

Comment:
out of curiosity, is there any reason to format the HTML if we're inserting it with JS?

Reply:
No, it would render fine even if there was no spaces. The spaces/tabs and newlines are there just so that it is easy to see the HTML and update it in the future.

Reply:
Oh, ok. Makes sense.